### DISCUSSION TITLE
Can we use Gutenberg editor separately?

### DISCUSSION THREAD

**Discussion by abhi3315** [WordPress/gutenberg#47860] [Developer Experience]
I was having an idea to design a plugin setting page which includes a Gutenberg editor.
Is there a way to include the Gutenberg editor outside the edit post-type pages?

**Comment by devfle** [comment]
Not sure if this is possible.. But one alternative could be initializing your own editor instance, wich is compatible with the Gutenberg Editor: https://github.com/getdave/standalone-block-editor

**Comment by abhi3315** [comment]
Thanks, @devfle. 🔥
